charts/stable/bitwardenrs/templates/deployment-ldapsync.yaml

```diff
@@ -30,7 +30,7 @@ spec:
           terminationMessagePolicy: File
           volumeMounts:
             - mountPath: /etc/bitwarden
-              name: {{ if .Values.ldapSync.existingSecret }}{{ .Values.ldapSync.existingSecret }}{{ else }}{{ include "bitwardenrs.name" . }}-ldap{{ end }}
+              name: {{ if .Values.ldapSync.existingSecret }}{{ .Values.ldapSync.existingSecret }}{{ else }}{{ .Values.ldapSync.configToml }}{{ end }}
```

Comment:
How does this work? `.Values.ldapSync.configToml` is a string like

```yaml
configToml: |
  [data]
  key=value
```

Reply:
Its a good point.  The config.toml file contents are normally base64 enc.  I am thinking that the existingSecret should be defined.  

So instead just this line in the config:

`{{ .Values.ldapSync.existingSecret }}`

Maybe you have a suggestion?

Reply:
Any update on this review please?

Reply:
We are all volunteers working on this when we can in our spare time. Please don't bump/tag so quickly, we will get to (re)reviewing when time allows us.

Reply:
> We are all volunteers working on this when we can in our spare time. Please don't bump/tag so quickly, we will get to (re)reviewing when time allows us.

Sorry about that, you are right ;)